packages/sx05re/emulators/sdlpop/package.mk

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2020-present Shanti Gilbert (https://github.com/shantigilbert)
+
+PKG_NAME="sdlpop"
+PKG_VERSION="12942c090ca653441c820b2abc8313ab748c0d24"
+PKG_ARCH="any"
+PKG_LICENSE="CUSTOM"
```

Comment:
According to the git for SDLpop this should be GPL3

Reply:
ok done